src/cclib/writer/ccwrite.py

```diff
@@ -28,7 +28,7 @@
 from . import xyzwriter
 
 
-def ccwrite(ccobj, outputtype=None, outputdest=None, returnstr=False,
+def ccwrite(ccobj, outputtype=None, outputdest=None, cjsonterse=False , returnstr=False,
```

Comment:
Maybe call this simply `terse` in this function which a comment that it only applies to CJSON at this point? Might be useful for other formats in the future.